### ISSUE TITLE
gt ltrclustering

### ISSUE THREAD

**Issue by ohan-Bioinfo** [genometools/genometools#906]
## Problem description

after the identification of LTR through Ltrharvest and annotation using Ltrdigest , I'm facing an error while trying to use (ltrsift) ltrclustering to group the output into families 

## Exact command line call triggering the problem

```
gt ltrclustering -psmall 80 -plarge 30 -o Out_clustring_ltrdigest index.fna Ltrdigst.gff
```

##Error output 

```
gt ltrclustering -psmall 80 -plarge 30 -o Out_Clustrer index.fna Ltrdigst_Result.gff 
Assertion failed: (eb->plainseq), function gt_encseq_builder_build, file src/core/encseq.c, line 9098.
This is a bug, please report it at
https://github.com/genometools/genometools/issues
Please make sure you are running the latest release which can be found at
http://genometools.org/pub/
You can check your version number with `gt -version`.
Abort trap: 6


```

## What GenomeTools version are you reporting an issue for (as output by `gt -version`)?
```
gt (GenomeTools) 1.5.10
Copyright (c) 2003-2016 G. Gremme, S. Steinbiss, S. Kurtz, and CONTRIBUTORS
Copyright (c) 2003-2016 Center for Bioinformatics, University of Hamburg
See LICENSE file or http://genometools.org/license.html for license details.

Used compiler: Apple LLVM version 10.0.0 (clang-1000.11.45.2)
Compile flags:  -g -Wall -Wunused-parameter -pipe -fPIC -Wpointer-arith -Wno-unknown-pragmas -O3 -m64 -Qunused-arguments -Wno-parentheses -Werror
```

## Did you compile GenomeTools from source? If so, please state the `make` parameters used.
yes

## What operating system (e.g. Ubuntu, Mac OS X), OS version (e.g. 15.10, 10.11) and platform (e.g. x86_64) are you using?
Mac OS Mojave version 10.14

**Comment by satta** [comment]
Thanks for reporting this! Can you please provide an example input set (sequence/index and GFF) for us to reproduce the problem?

**Comment by satta** [comment]
Thanks, but I'm afraid I will need both complete input files. The sequence provided by you above looks a bit short (and in a strange format) and the GFF consists of only one element with coordinates that do not seem to match the sequence length.

**Comment by bioinfo-Kacst** [comment]
i just toke part of the sequence, should i share all of it here (copy&past) or its possible to attcha it here @satta

**Comment by satta** [comment]
If that's OK with you, you could use a file hosting service such as Dropbox etc. to provide the files or drag & drop them into the comment box to upload them here.. Please avoid pasting long content here.

**Comment by bioinfo-Kacst** [comment]
https://drive.google.com/open?id=1_AADEBXvCZCNqMFRXPUcdafFwrpQq25d
could try this link if its working @satta

**Comment by satta** [comment]
Yes, it seems to work. I will need some time to look into the issue though...

**Comment by bioinfo-Kacst** [comment]
Dear @satta current step will classify and conclude my ltr through gt ltrclustering? or further analysis involves.
kindly I'm stuck in this bug would you please help
